manager/src/main/resources/application.properties

```diff
@@ -45,3 +45,22 @@ quarkus.native.resources.includes=exception/exceptionInfo.json
 %dev.quarkus.iam.aws.credentials.type=static
 %dev.quarkus.iam.aws.credentials.static-provider.access-key-id=testing
 %dev.quarkus.iam.aws.credentials.static-provider.secret-access-key=testing
+
+
+# The Kafka broker location (defaults to localhost:9092)
+kafka.bootstrap.servers=${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
```

Comment:
This is wrong in general: we should use MK API (similar to what we do for now with MC) to get Kafka information.
In the future we might have also multiple instances and the manager decide where to deploy
Do we have a similar ticket?

Reply:
https://issues.redhat.com/browse/MGDOBR-197